examples/mini-farmer.rs

```diff
@@ -116,20 +116,23 @@ async fn main() -> anyhow::Result<()> {
         )
         .await?;
 
-    let subscriptions = {
-        let plot = farmer.iter_plots().await.next().unwrap();
-
-        let node = &node;
-
+    tokio::spawn({
```

Comment:
here (the previous comment)

Reply:
I think it is fine. We just spawn an initial progress tracker in the background. So if we don't await on it, it will still cancel everything at exit.